trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'base_decorator'
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    trimmed_name = @nameable.correct_name[0...9]
+    trimmed_name.length == @nameable.correct_name.length ? trimmed_name : "#{trimmed_name}..."
```

Comment:
- Trimmer needs to return this `Maximilan` not Maximila, kindly check into this as well
![image](https://github.com/Metaverse-Nabeel/OOP-school-library/assets/62705947/7308192c-2eed-4006-8694-8bb5ee10672c)

Reply:
I have made the changes and now it works fine.

Reply:
Hello @GzimAsani 

Now, it is working fine.